public/images/flags/ERI.svg

```diff
@@ -1,6 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" id="flag-icons-er" viewBox="0 0 640 480">
```

Comment:
Flagging (ha) all the weird colours I spot so we can maybe do a batch update in an upstream PR